x-pack/plugins/features/server/oss_features.ts

```diff
@@ -260,6 +261,7 @@ export const buildOSSFeatures = ({ savedObjectTypes, includeTimelion }: BuildOSS
                     read: [],
                   },
                   ui: ['storeSearchSession'],
+                  api: ['store_search_session'],
```

Comment:
Any reason why one is camel case and one is snake case?

Reply:
I don't know why `ui` is camel, but the id of the privilege itself is lowercase, I decided to go with lowercase because the main id is lowercase 🤷‍♂️ . not sure if there is a convention 
https://github.com/elastic/kibana/blob/ee1b71591e8022f6bc2f85fa2ab532aedf7a5463/x-pack/plugins/features/server/oss_features.ts#L85